alpaca/broker/client.py

```diff
@@ -1,45 +1,58 @@
-from enum import Enum
-from typing import Callable, Iterator, List, Optional, TypeVar, Union
-from uuid import UUID
-from pydantic import parse_obj_as
 from itertools import chain
+from typing import Callable, Iterator, List, Optional, Union
+from uuid import UUID
 
-from ..common.constants import ACCOUNT_ACTIVITIES_DEFAULT_PAGE_SIZE
-from ..common import APIError
-from ..common.enums import BaseURL, PaginationType
-from ..common.rest import HTTPResult, RESTClient
-from ..common.models import BaseActivity, TradeActivity, NonTradeActivity
+from pydantic import parse_obj_as
+from requests import HTTPError, Response
 
+from .constants import BROKER_DOCUMENT_UPLOAD_LIMIT
 from .models import (
     Account,
     AccountCreationRequest,
     AccountUpdateRequest,
+    ActivityType,
     CIPInfo,
+    GetAccountActivitiesRequest,
+    GetTradeDocumentsRequest,
     ListAccountsRequest,
     TradeAccount,
-    GetAccountActivitiesRequest,
-    ActivityType,
+    TradeDocument,
+    UploadDocumentRequest,
 )
+from ..common import APIError
+from ..common.constants import ACCOUNT_ACTIVITIES_DEFAULT_PAGE_SIZE
+from ..common.enums import BaseURL, PaginationType
+from ..common.models import BaseActivity, NonTradeActivity, TradeActivity
+from ..common.rest import HTTPResult, RESTClient
 
 import base64
 
 
-def validate_account_id_param(account_id: Union[UUID, str]) -> UUID:
+def validate_uuid_id_param(
+    account_id: Union[UUID, str],
```

Comment:
Maybe we can rename to `account_id` to `id` to reflect that it is a generic uuid.

Reply:
oh for sure cant believe I missed that rename :see_no_evil: